rfcs/0043-rfcsc-rotation.md

```diff
@@ -0,0 +1,104 @@
+---
+feature: rfcsc-rotation
+start-date: 2019-04-24
+author: Robin Gloster <mail@glob.in>, Simon Lackerbauer <mail@ciil.io>
+related-issues: 36
+---
+
+# Summary
+[summary]: #summary
+
+Each RFC Steering Committee (RFCSC) unanimously elects the succeeding one at
+their first meeting in December from an open list of nominees.
+
+# Motivation
+[motivation]: #motivation
+
+The RFC Steering Committee has been established by [RFC
+36](https://github.com/NixOS/rfcs/blob/master/rfcs/0036-rfc-process-team-amendment.md).
+Future work for that RFC included a definition of how members to the Committee
+are to be chosen or removed. This RFC provides mechanisms for beginning and
+ending Steering Committee membership. The purpose of this is to ensure that the
+committee:
+
+ * continues to consist of well-informed and motivated members;
+ * continues to represent the evolving community;
+ * remains active and keeps the RFC process running.
+
+# Detailed design
+[design]: #detailed-design
+
+The RFC Steering Committee shall always have five members. If membership drops
+below five members (for example by resignation of a member as detailed below),
+a new member shall be elected without delay after a nomination period of at
+least two weeks (see below for nomination and selection process). If the number
+of members of the RFCSC drops below 4 people, it cannot proceed with shepherd
+team selections until new members have been selected.
+
+## Ending membership
+A member can end their membership in the Steering
+Committee by either of the following four mechanisms:
```

Comment:
Maybe rephrase as «Membership of a member in the Steering Committee can end by …»

(I _tried_ to do a two-line suggestion and failed)